automation/features/pageobjects/vesting.page.js

```diff
@@ -0,0 +1,33 @@
+import Page from "./page";
+
+class VestingPage extends Page {
+  get trancheItemTestId() {
+    return "tranche-item";
+  }
+  get validatorNodeTitle() {
+    return $('data-test-id="validator-node-title"]');
```

Comment:
Missing a square bracket here. Probably worth checking any tests here to make sure they aren't evergreen

Reply:
veryyyyyyy observant thanks !